sim/rogue/poisons.go

```diff
@@ -148,6 +152,10 @@ func (rogue *Rogue) applyDeadlyBrewDeadly() {
 				rogue.DeadlyPoison[DeadlyBrewProc].Cast(sim, result.Target)
 			}
 		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			fmt.Println("Deadly Brew (Deadly) deactivated")
+			rogue.PoisonsActive--
+		},
```

Comment:
This aura is a bit missleading but it's actually just acting as a proc trigger for deadly brew, not an actual poison effect